inc/functions_online.php

```diff
@@ -516,27 +516,21 @@ function fetch_wol_activity($location, $nopermission=false)
 			{
 				$user_activity['activity'] = "usercp_subscriptions";
 			}
-			elseif($parameters['action'] == "notepad" || $parameters['action'] == "do_notepad")
-			{
-				$user_activity['activity'] = "usercp_notepad";
-			}
-			else
+			elseif($parameters['action'] == "addfavorite" || $parameters['action'] == "removefavorite" || $parameters['action'] == "removefavorites")
```

Comment:
Do we have favorites in mybb ?

Reply:
They were removed back in 1.4.

Reply:
Might as well leave it there if it's currently in `usercp2.php` - this should just be a direct port. We can remove dead code like that at a later date.